git-contrib-graph.go

```diff
@@ -167,7 +177,28 @@ func printAuthors(contribs map[string]map[string]Stats) {
 			getPlusMinusProgression(0, deletionSum, NBR_COLUMN-20),
 		)
 
-		printAuthorContribGraph(minDate, maxDate, days)
+		printAuthorContribGraph(minDate, maxDate, days, &perDay)
+
+		if JSON_OUTPUT {
+			out = fmt.Sprintf(
+				out+`{"autor": "%s", "total": {"commits": %d, "insertions": %4d, "deletions": %4d}, "per_day": %s},`,
+				author,
+				commitCount,
+				additionSum,
+				deletionSum,
+				perDay,
+			)
+		}
+	}
+	if JSON_OUTPUT {
+		// remove last "," character from json
```

Comment:
you can do it with `strings.Join(",")` ;-)

Reply:
by making this an array i'm no longer need slice this last coma